test/models/like_test.rb

```diff
@@ -1,4 +1,4 @@
-require "test_helper"
+require 'test_helper'
 
```

Comment:
Great job done so far @julzedz 
Please permit me to use this line to request changes to this Pull Request since your code dose not reflect on the PR.

- [ ] Could you fix the error on line 5 in your likes migration file?  is `t.references :post, null: false, foreign_key: true`.

**Not belongs_to**
<img width="615" alt="Screenshot 2023-07-14 at 04 28 37" src="https://github.com/julzedz/tech-krunch/assets/55596970/c20d8d4f-f925-4c1a-8ac2-3473f356cac7">